src/Import/IcsImport.php

```diff
@@ -137,7 +137,7 @@ public function importFromIcsFile(Vcalendar $cal, CalendarModel $objCalendar, Da
                 $dtend = $vevent->getDtend(true);
 
                 $rrule = $vevent->getRrule();
-                $summary = $vevent->getSummary() ?? '';
```

Comment:
The title can be `null` for example from a google calendar but the title cannot be `null` in the event and a empty title `''` will not be possible for the event reader because of errors with `JSON-LD`